.github/workflows/docker-git.yml

```diff
@@ -38,7 +48,6 @@ jobs:
         run: |
           docker run --privileged --rm tonistiigi/binfmt --install arm64,amd64
           docker buildx create --use --name cross-builder
-      - name: Build and push the git-sha-tagged reth image
-        run: make PROFILE=maxperf GIT_SHA=$GIT_SHA docker-build-push-git-sha
-      - name: Build and push the git-sha-tagged op-reth image
-        run: make IMAGE_NAME=$OP_IMAGE_NAME DOCKER_IMAGE_NAME=$OP_DOCKER_IMAGE_NAME GIT_SHA=$GIT_SHA PROFILE=maxperf op-docker-build-push-git-sha
+      - name: Build and push the git-sha-tagged ${{ matrix.image-name }} image
+        run: |
+          CI=true make IMAGE_NAME=${{ matrix.image-name }} DOCKER_IMAGE_NAME=${{ matrix.docker_image_name }} GIT_SHA=$GIT_SHA PROFILE=maxperf ${{ matrix.make_target }}
```

Comment:
you can make this nicer by setting `env` in the job itself (not the workflow) based on the matrix values